user-guide/Standard_Apps/Monitoring_Solutions/DITT/Working_With_DITT.md

```diff
@@ -0,0 +1,38 @@
+---
+uid: Working_With_DITT
+---
+
+# Implementing DITT in a visual overview
+
+It is possible to implement DITT in a visual overview of your own:
```

Comment:
What would be the advantage of implementing DITT in a different visual overview? Could you add this at the top of the page so it's clear for a user why they should follow the procedure below?